phac-nml/irida-next
# Fix broken seeds.rb file and add logging

## What does this PR do and why?
Fixed the crashing `seeds.rb` file when run with `rake db:seed` because of the duplicate file `test/fixtures/files/test_file_D.fastq`
Fixes: #299

Added logging to the seed rake task

Added option to output logging for all rake tasks

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._
N/A

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._
1. Run `rake db:drop db:create db:migrate db:seed`, and verify the data has been seeded
2. Run `rake info db:drop db:create db:migrate db:seed` (or `./scripts/clean_dev.sh`, and see the new logging

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
